README.md

```diff
@@ -33,16 +33,26 @@ These prefixes, like the rest of the file name, have no semantic meaning and jus
 Each file contains the following in [YAML format](https://en.wikipedia.org/wiki/YAML):
 
 - A `name` (typically mirrored in the file name), this is also the `artifactId` of the Maven artifact.
+- A `github` field indicating the GitHub organization and repository which is expected to produce these artifacts.
```

Comment:
Add *(optional)*

Reply:
_Currently_ it is optional. I would like to make it mandatory, as soon as I have time to write up a script to scan all POMs and the like in a @jenkinsci checkout. Marking it optional would indicate to people submitting new files that they could skip this step, when I would want them to include the field.

Reply:
You could re-use the `scm` attributes in http://updates.jenkins-ci.org/update-center.actual.json (which is also where the plugin site gets its GitHub links).

Not always present because POMs can be surprisingly terrible, but already does some guesswork in https://github.com/jenkins-infra/update-center2/blob/91c94226b2f9b3428919ac929b33b76b2139a8d1/src/main/java/org/jvnet/hudson/update_center/Plugin.java#L335 and related code.

Reply:
Thanks, that is a good tip.